### ISSUE TITLE
Precious lint --staged fails when all files are excluded

### ISSUE THREAD

**Issue by horgh** [houseabsolute/precious#34]
I'm using precious 0.2.2. I'm trying to lint with only a file that falls under `exclude` and I receive an error.

A sample config looks like this:

```toml
exclude = [
    "**/testdata/**/*",
]

[commands.golangci-lint]
type = "both"
cmd = [ "golangci-lint", "run", "." ]
chdir = true
tidy_flags = [ "--fix" ]
ok_exit_codes = 0
expect_stderr = true
run_mode = "dirs"
include = [ "go/**/*.go" ]
```

```
$ git status
On branch test-precious
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   go/dev/xgb2code/testdata/main.go
```

But I get:

```
$ precious lint --staged
💍 Linting files staged for a git commit
[precious_core::precious][ERROR] Failed to run precious: Found some paths when looking for files staged for a git commit but they were all excluded
```

What do you think?

Thank you!

**Comment by autarch** [comment]
I think in `--staged` and `--git` mode it shouldn't return this error, since it's likely to break pre-commit hooks. But I'll leave this error as-is in other modes.

**Comment by autarch** [comment]
Fixed in v0.2.3
